express-get-json/server.ts

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+
+const app = express();
+const port = 8080;
+
+type Grade = {
+  id: number;
+  name: string;
+  course: string;
+  score: number;
+};
+
+const grades: Record<number, Grade> = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Håkon Wium Lie',
+    course: 'CSS',
+    score: 92,
+  },
+};
+
+app.get('/api/grades', (req, res) => {
+  console.log('GET req running'); // remove later
+  res.json(grades);
```

Comment:
The instructions tell you to respond with an array, not an object.

Reply:
<img width="305" alt="Screenshot 2023-10-20 at 11 52 15 AM" src="https://github.com/jeremyhnguyen/c-0823-code-solutions/assets/136545701/915de50f-82fa-4ecf-b829-34b688ed7d41">